karma.conf.js

```diff
@@ -135,9 +135,10 @@ module.exports = function (config) {
         ],
 
         customLaunchers: {
-            'ChromeHeadless': {
+            'VectorChromeHeadless': {
```

Comment:
Do we still use the `Vector` prefix that much, now that it's called Riot? Also, what's the purpose of the custom name here in any case?

Reply:
I'm using Vector in the company sense here, not in the product sense. `ChromeHeadless` is an actual browser version and seems to confuse Buildkite (but not Travis CI for some reason), so I've essentially named it `New Vector's ChromeHeadless Browser v10: Spectacular feature release for CI environments` without using so many words.